pyproject.toml

```diff
@@ -63,6 +63,9 @@ mkdocstrings-python = "0.7.1"
 [tool.poetry.extras]
 metrics = ["nautobot-capacity-metrics"]
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
I guess this comes from some new poetry version, could you group this with the other dev dependencies?